CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- typo in Receive grant payment cert task action - certifcate to certificate
```

Comment:
```suggestion
- typo in Receive grant payment certificate task action fixed - certifcate 
  becomes certificate
```

Reply:
You should be able to do this to silence the linter error - I think the line was just too long.